js/views/storeWizardVw.js

```diff
@@ -148,12 +150,19 @@ module.exports = baseModal.extend({
   },
 
   renderModerator: function(moderator){
+    //make sure this moderator is not a duplicate
```

Comment:
The dupes are in there because the server sends duplicate data. 

We could move the check to the socket message handler.